ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -89,12 +90,13 @@ data IsProducer
 run
   :: forall blk.
      RunNode blk
-  => Tracers IO ConnectionId blk          -- ^ Consensus tracers
-  -> Tracer  IO (ChainDB.TraceEvent blk)  -- ^ ChainDB tracer
-  -> DiffusionTracers                     -- ^ Diffusion tracers
-  -> DiffusionArguments                   -- ^ Diffusion arguments
+  => Tracers IO ConnectionId blk                             -- ^ Consensus tracers
+  -> ProtocolTracers IO ConnectionId blk DeserialiseFailure  -- ^ Protocol tracers
```

Comment:
Nitpick: this line is now longer 80 chars. Suggestion:
```haskell
  => Tracers IO ConnectionId blk          -- ^ Consensus tracers
  -> ProtocolTracers IO ConnectionId blk DeserialiseFailure
     -- ^ Protocol tracers
  -> Tracer  IO (ChainDB.TraceEvent blk)  -- ^ ChainDB tracer
```
You can revert the changes that realign the Haddock comments.